src/main/java/com/rtb/projectmanagementtool/user/CreateUserServlet.java

```diff
@@ -86,4 +72,18 @@ else if (userService.isUserLoggedIn()) {
     // redirect them to their new profile
     response.sendRedirect("/user_profile.jsp");
   }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get login URL
+    String urlToRedirectToAfterUserLogsIn = "/create-new-user";
+    String loginUrlNewUser = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+
+    // send to request
+    request.setAttribute("loginUrlNewUser", loginUrlNewUser);
+    request.getRequestDispatcher("login.jsp").forward(request, response);
```

Comment:
ERROR: CreateUserServlet.java:[84,54] unreported exception javax.servlet.ServletExcept
ion; must be caught or declared to be thrown

Reply:
Add ServletException to line 77 and import it.